libs/services/designer-client-services/src/lib/standard/run.ts

```diff
@@ -239,11 +241,14 @@ export class StandardRunService implements IRunService {
    * @returns {BoundParameters} List of parametes.
    */
   parseActionLink(response: Record<string, any>, isInput: boolean): BoundParameters {
-    if (!response) {
+    if (isNullOrUndefined(response)) {
       return response;
     }
 
-    const dictionaryResponse = isString(response) ? { [isInput ? 'Inputs' : 'Outputs']: response } : response;
+    const dictionaryResponse =
```

Comment:
Array should not be handled as object?

Reply:
What about isBoolean

Reply:
You are right, I will add the isBoolean. I handed the array as this because because it wasn't showing as an object, so I put it in an inputs/outputs object.  You can see the example below

- Before
<img width="300" alt="image" src="https://github.com/Azure/LogicAppsUX/assets/102700317/96f01e42-3956-4923-9a9b-dbd282bf6bdb">

- After
<img width="300" alt="image" src="https://github.com/Azure/LogicAppsUX/assets/102700317/9d06c400-141d-46fb-9a0a-ee81dc203bcc">